packages/graphql/lib/factories/context-id-factory.ts

```diff
@@ -0,0 +1,28 @@
+import { ContextId, createContextId } from '@nestjs/core';
+import { REQUEST_CONTEXT_ID } from '@nestjs/core/router/request/request-constants';
+
+export abstract class ContextIdFactory {
```

Comment:
The reason why I use an abstract class here instead of an interface is to be able to use it as an Injection Token.